portfolio.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="portfolio.css"><script src="https://kit.fontawesome.com/0a62dabd9f.js" crossorigin="anonymous"></script>
+
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    
+    <header><h1>Hey there. I’m Cleophas.</br>
+        <div class="green">I’m a software developer</div>
+        </h1></header>
+    <main>
+        
+        <section>
+            <p> I can help you build a product , feature or website Look through some of my work and </br>
+                 experience! If you like what you see and have a project you need coded, </br>
+                don’t hestiate to contact me.</p>
+        </section>
+    </main>
+    <footer class="social">
+       <a href="https://twitter.com/bettcleos"><i class="fa-brands fa-twitter"></i></a> 
+        <a href="https://www.linkedin.com/in/kiprono-cleophas-026a62244/"><i class="fa-brands fa-linkedin"></i></a>
+       <a href="https://github.com/bettcleos"><i class="fa-brands fa-github"></i></a> 
+        <a href= ""><i class="fa-solid fa-m"></i></a>
+        
+    </footer>
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Kindly look into these and fix them.
Something like this. :point_down: 
```
    <ul>
         <li>
              <a href="#">Your icon</a>
         </li>
          <li>
              <a href="#">Your icon</a>
         </li>
     </ul>
```